processes-endpoint.md

```diff
@@ -0,0 +1,107 @@
+# Admin Processes Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+This endpoint allows adminstrators to list and manipulate script processes created by the [Scripts endpoint](scripts-endpoint.md).
+
+## Execution List
+**GET /api/admin/processes**
```

Comment:
I don't like the admin category as it seems to enforce authz that can evolve over time. Maybe we can use "run" ?

Reply:
Or maybe "system"?